packages/@glimmer/runtime/lib/compiled/opcodes/debugger.ts

```diff
@@ -19,6 +19,22 @@ function debugCallback(context: unknown, get: DebugGet): void {
   debugger;
 }
 
+function templateOnlyDebugCallback(get: DebugGet): void {
+  console.info('Use `get(<@path>)` to debug this template.');
+
+  // for example... `get('@foo');`
+  // eslint-disable-next-line no-debugger
+  debugger;
+}
```

Comment:
```
$ tsc -p tsconfig.json --noEmit --pretty false
packages/@glimmer/runtime/lib/compiled/opcodes/debugger.ts(22,36): error TS6133: 'get' is declared but its value is never read.
```

Reply:
@rwjblue yes I notice this error, but I’m not sure what’s the right way to fix this. Could you please help in here?

Reply:
You can probably disable it manually, but without actually calling `get` it is likely for the JS runtime to optimize it away (making this not work properly).

If you take a look at the other pathway, we do `get(this)` so that the `get` function is used...

Reply:
I've added a variable assignment and ignored using `ts-ignore`. Should we do it this way or say `console.log(get('@foo'));`? The console output would output undefined all the time someone tries to use a `debugger` on the template.

Reply:
Ya, I think in general what you have is good, though I wonder if we can silence the TS error either globally for that file or some ohter way (basically I'd prefer the user not see `//@ts-ignore` in the code just above their debugger).